plugin/updater_oneapi/README.md

```diff
@@ -0,0 +1,36 @@
+# DPC++-based Algorithm for Tree Construction
```

Comment:
Please add a notice that the parameter name might change in the future, as we want to have a more integrated experience as time move.

Reply:
Done.